prow/github/client.go

```diff
@@ -819,6 +849,12 @@ func (c *Client) ReopenPR(org, repo string, number int) error {
 		requestBody: map[string]string{"state": "open"},
 		exitCodes:   []int{200},
 	}, nil)
+	requestErr, ok := err.(requestError)
+	if ok && strings.Contains(requestErr.Message, "state cannot be changed.") {
```

Comment:
nit: it'd be good to define "state cannot be changed." as a const string in case it changes in the future.

Reply:
done